### PR TITLE
Aaron/merged protected routes

### PR DESCRIPTION
**Pull request by ethanstrominger**

### CODE REVIEW

**Comment by ethanstrominger** [comment]
Aaron - Can you make two changes:
- To check if user is logged in, check if get_current_user().username exists.  I recommend creating a function is_logged_in in authorize.py for this.
- Get rid of console.log('here')

File: src/App.js [hunk 1]
```diff
@@ -1,72 +1,33 @@
...
-      </header>
+      <Header />
 
-      <main className="flex-shrink-0" role="main">
+      <main>
         <div className="container mt-3">
           <Switch>
             <Route exact path={["/", "/orders"]} component={OrdersList} />
-            <Route exact path="/orders/add" component={AddOrder} />
+            <PrivateRoute exact path="/add">
```
Comment (rkaufman13) [inline src/App.js:23]: Did we change the path to not be /orders/add?

File: src/App.js [hunk 2]
```diff
@@ -1,72 +1,33 @@
...
+              <AddOrder />
+            </PrivateRoute>
             <Route path="/orders/:id" component={EditOrder} />
```
Comment (rkaufman13) [inline src/App.js:26]: edit should probably also be private

File: src/components/privateHeader.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+              <Link to={"/orders"} className="nav-link">
+                Orders
+              </Link>
+            </li>
+            <li className="nav-item">
+              <Link to={"/add"} className="nav-link">
+                Add
+              </Link>
+            </li>
+            <li className="nav-link">Logout</li>
```
Comment (rkaufman13) [inline src/components/privateHeader.js:23]: The original version of the header had a link to "profile" page - that should still be available here when user is logged in

File: package.json [hunk 4]
```diff
@@ -2,8 +2,7 @@
   "name": "us-congress-pizza-flag-tracker-frontend",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:5000/",
```
Comment (rkaufman13) [inline package.json:5]: do we not need this anymore?

**Review by rkaufman13 — COMMENTED** [review]
looks good! did we decide against the /profile page though?

**Pull request closed without merging**